### ISSUE TITLE
This is the most difficult plug-in I've ever used

### ISSUE THREAD

**Issue by aaTking** [CodeCrafter47/BungeeTabListPlus#298]
Especially the playerSet part
I just want to hit people
Can I beat you up, writer?

**Comment by CodeCrafter47** [comment]
You're right there, the player set part is a bit difficult to understand. Have you checked the wiki: <https://github.com/CodeCrafter47/BungeeTabListPlus/wiki/Configuration-Basics-Part-4---Player-sets>?
